resources/metadata/xsDeploy.yaml

```diff
@@ -0,0 +1,128 @@
+metadata:
+  name: xsDeploy
+  description: Performs xs deployment
+  longDescription: |
+    Performs xs deployment
+
+spec:
+  inputs:
+    resources:
+    params:
+      - name: deployOpts
+        type: string
+        description: Additional deploy options
+        scope:
+        - GENERAL
```

Comment:
not sure if GENERAL really makes sense for this parameter, I suggest to use it only selectively ...